blog-home.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog-home
+template-title: Blog Home
+template-description: Card-based landing page for a blog with media cards, tags, and secondary navigation
+title: Blog
+description: Latest updates on how agencies are serving the American people
+source-domain: usa.gov
+source-url: tbd
+blog-posts:
+    - title: "How USAGov Cracked a Cybercrime Using Free Web Analytics Tools"
+      author: "Sondra Ainsworth and Constance Lu"
+      date: "September 30, 2020"
+      image: https://raw.githubusercontent.com/Bixal/rrt-content/main/assets/img/blog-4.jpg
+      labels:
+        - name: Federal
+        - name: Politics
+        - name: Technology
+        - name: Public Service
+    - title: "Partner Spotlight: Informing the Public About the 2020 Census"
+      author: "Sondra Ainsworth and Constance Lu"
+      date: "August 30, 2020"
+      image: https://raw.githubusercontent.com/Bixal/rrt-content/main/assets/img/blog-1.jpg
+      labels:
+        - name: Public
+        - name: Politics
+        - name: Federal
+    - title: "Making it Easier for People to find Federal Agencies"
+      author: "Sondra Ainsworth and Constance Lu"
+      date: "July 30, 2020"
+      image: https://raw.githubusercontent.com/Bixal/rrt-content/main/assets/img/blog-2.jpg
+      labels:
+        - name: Federal
+        - name: Politics
+        - name: News
+    - title: "USAGov’s Guide to Instagram"
+      author: "Sondra Ainsworth and Constance Lu"
+      date: "June 30, 2020"
+      image: https://raw.githubusercontent.com/Bixal/rrt-content/main/assets/img/blog-3.jpg
+      labels:
+        - name: Federal
+        - name: Technology
+        - name: Social
+blog-categories:
```

Comment:
Now that I've seen the source page, I understand why you had tags initially (because they're not using categories). In that case, I would change the heading of this side nav to Tags. And let's use their content so we don't have things like "Plants" and "Farming" that seem out of place. Also, it will make this list shorter, which avoids us having to figure out how we might collapse it if it gets really long (at least for now).